blobstore/common/rpc/auditlog/auditlog.go

```diff
@@ -127,68 +143,48 @@ func (j *jsonAuditlog) Handler(w http.ResponseWriter, req *http.Request, f func(
 	f(_w, req)
 
 	endTime := time.Now().UnixNano() / 1000
-	b := j.logPool.Get().(*bytes.Buffer)
-	defer j.logPool.Put(b)
-	b.Reset()
-
-	b.WriteString("REQ\t")
-	b.WriteString(j.module)
-	b.WriteByte('\t')
-
-	// record request info
-	b.WriteString(strconv.FormatInt(startTime/100, 10))
-	b.WriteByte('\t')
-	b.WriteString(req.Method)
-	b.WriteByte('\t')
-	b.WriteString(decodeReq.Path)
-	b.WriteByte('\t')
-	b.Write(decodeReq.Header.Encode())
-	b.WriteByte('\t')
-	if len(decodeReq.Params) <= maxSeekableBodyLength && len(decodeReq.Params) > 0 {
-		b.Write(decodeReq.Params)
-	}
-	b.WriteByte('\t')
 
-	// record response info
-	respContentType := _w.Header().Get("Content-Type")
-	b.Write(_w.getStatusCode())
-	b.WriteByte('\t')
-
-	// Check if track-log and tags changed or not,
-	// if changed, we should set into response header again.
-	// But the additional headers DO NOT write to client if
-	// they set after response WriteHeader, just logging.
-	wHeader := _w.Header()
-	traceLogs := span.TrackLog()
-	if len(wHeader[rpc.HeaderTraceLog]) < len(traceLogs) {
-		wHeader[rpc.HeaderTraceLog] = traceLogs
-	}
-	tags := span.Tags().ToSlice()
-	if len(wHeader[rpc.HeaderTraceTags]) < len(tags) {
-		wHeader[rpc.HeaderTraceTags] = tags
```

Comment:
`traceLogs` and `traceTags` MUST fill into response headers. Cos the HTTP Handler will set more traces to `span`, not the response object.